main-targets.mk

```diff
@@ -58,7 +58,7 @@ docker: ## Build a Docker image
 ifneq (${DOCKER_PREBUILT}, 1)
 	@${MAKE} BINARY_NAME="${BINARY_NAME}-linux-amd64" GOOS=linux GOARCH=amd64 build-release
 endif
-	docker build --build-arg BUILD_DIR=${BUILD_DIR}/release --build-arg BINARY_NAME=${BINARY_NAME}-linux-amd64 -t ${DOCKER_IMAGE}:${DOCKER_TAG} -f Dockerfile.local .
+	docker build --build-arg BUILD_DIR=${BUILD_DIR}/release --build-arg BINARY_NAME=${BINARY_NAME}-linux-amd64 -t ${DOCKER_IMAGE}:${DOCKER_TAG} -f Dockerfile .
```

Comment:
Why docker file name has to be changed ?

Reply:
there is no such (Dockerfile.local) dockerfile in the project; the make target fails